pom.xml

```diff
@@ -31,7 +31,7 @@
         </argLine>
 
         <aws.version>1.11.538</aws.version>
-        <jackson.version>2.9.6</jackson.version>
+        <jackson.version>2.8.3</jackson.version>
```

Comment:
Why `2.8.3`? Why not latest `2.11.0`? Or `2.8.11.20200310`, latest from `2.8.x`? What's the reasoning be behind previous `2.5.1` or this `2.8.3` specific versions?

Reply:
`2.5.1` is the version on glassfish, `2.8.3` is the lowest version that compiles and works, I could try each version to find the highest one but the higher version the higher the chance that it will conflict with glassfish version.

Reply:
Ah, now I understand. In that case, `2.5.1` in `provided` scope seems the way to go. Since there is no `BOM` for that version, I'd remove it entirely and modify `<dependencyManagement> section with specific dependencies.